lib/appraisal/path_source.rb

```diff
@@ -1,30 +1,20 @@
-require 'appraisal/dependency_list'
+require 'appraisal/bundler_dsl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.